apps/api/src/environment/service/environment.service.ts

```diff
@@ -195,11 +196,29 @@ export class EnvironmentService {
         }
       },
       skip: page * limit,
-      take: limit,
+      take: Number(limit),
```

Comment:
The limit field was already parsed into number. Not sure why this step was required.

Reply:
As I told you already on discord. This was failing in e2e tests, so had to explicitly do it.

Reply:
Important one @rajdip-b  da. The QueryTransformPipe only works everywhere except during unit and e2e tests. So page and limit are always string, when value is explicitly specified.
The issue is caused due to app being not initialized with globalPipes during tests.
Doing this solves the issue: 
![image](https://github.com/user-attachments/assets/2fc9b69e-f555-4e93-9420-4519d10abfc8)

Now, how should i proceed??

Reply:
I got this approach.
- Use number conversion as of now
- Modify e2e and unit along with removing those number conversion with a new PR.

Reply:
Yes! I was guessing this might be the issue but didnt get time to look into it. If this fix is working, then I would ask you to add the useGlobalPipes thing to the test suite and then run the tests.

Reply:
I'll make the final review once you give me an update on this!

Reply:
I have pushed required changes. Kindly, check.